### PR TITLE
Remove java "-mx128m" option

### PR DESCRIPTION
**Pull request by aureocarneiro**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by aureocarneiro** [comment]
@conda-forge-admin, please rerender

File: recipe/build.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@ fi
 export CLASSPATH=${PREFIX}/share/java/Jive.jar
 
 java \
-    -mx128m \
+    -mx1g \
```
Comment (beenje) [inline recipe/build.sh:26]: We should probably just remove this setting and keep the default.
If we don't pass it, we can as well use env variable to change the value.

This comes from https://gitlab.com/tango-controls/TangoSourceDistribution/-/blob/main/assets/lib/java/jive.in?ref_type=heads#L24

We should open an issue there and do the same thing.

**Comment by aureocarneiro** [comment]
@conda-forge-admin, please rerender

File: recipe/build.sh (hunk 1 above)
Comment (aureocarneiro) [reply recipe/build.sh]: updated

Comment (beenje) [reply recipe/build.sh]: Thanks. Forgot to mention you have to update `jive.bat` as well for windows.

Comment (aureocarneiro) [reply recipe/build.sh]: done

**Pull request merged**